fpga_interchange/family.cmake

```diff
@@ -5,8 +5,26 @@ endif()
 
 find_package(ZLIB REQUIRED)
 
+set(RAPIDWRIGHT_PATH $ENV{HOME}/RapidWright CACHE PATH "Path to RapidWright")
+set(INVOKE_RAPIDWRIGHT ${RAPIDWRIGHT_PATH}/scripts/invoke_rapidwright.sh CACHE PATH "Path to RapidWright invocation script")
+# FIXME: Make patch data available in the python package and remove this cached var
+set(PYTHON_INTERCHANGE_PATH $ENV{HOME}/python-fpga-interchange CACHE PATH "Path to the FPGA interchange python library")
+set(INTERCHANGE_SCHEMA_PATH $ENV{HOME}/fpga_interchange_schema CACHE PATH "Path to the FPGA interchange schema dir")
+
 add_subdirectory(3rdparty/fpga-interchange-schema/cmake/cxx_static)
 
+include(${family}/examples/chipdb.cmake)
+include(${family}/examples/tests.cmake)
+
+set(chipdb_dir ${CMAKE_CURRENT_BINARY_DIR}/${family}/chipdb)
+file(MAKE_DIRECTORY ${chipdb_dir})
+
+set(root_dir ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Why is `root_dir` needed?  This seems like a bad idea?

Reply:
This was only required to get the location of the `check_arch_api.py` for the archcheck target. Maybe I can find another way to get that script

Reply:
In that case, you should be using `${PROJECT_SOURCE_DIR}` or `${nextpnr_SOURCE_DIR}`.

Reply:
Done